designer/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentsUsageHelper.scala

```diff
@@ -12,21 +13,24 @@ object ComponentsUsageHelper {
 
   def computeComponentsUsageCount(
       processesDetails: List[ScenarioWithDetailsEntity[ScenarioComponentsUsages]],
-      processingTypeAndComponentInfoToComponentId: (ProcessingType, ComponentInfo) => ComponentId,
+      processingTypeToNonFragmentComponents: Map[ProcessingType, Map[ComponentInfo, ComponentStaticDefinition]]
```

Comment:
I know that I proposed similar approach during the previous review, but I see know that this signature looks ugly. can you flatten this map to `Map[(ProcessingType, ComponentInfo), ComponentId]?`

Reply:
I like it, let it stay